packages/opds-web-client/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog
 
+### v0.3.4
```

Comment:
This is fine but it may not match one-to-one with the changes that can potentially be added in `master`. Not a big deal but just something to keep in mind.

Reply:
Okay yep I can change to whatever is needed